start-training

```diff
@@ -40,16 +40,20 @@ parser.add_argument('-p', '--impact-penalty', action='append', type=float)
 parser.add_argument('--penalty-baseline',
     choices=('starting-state', 'inaction'), default='starting-state')
 parser.add_argument('-e', '--env-type', default='prune-still')
+parser.add_argument('--curriculum', default="progress_estimate", type=str,
+    help='Curriculum type ("uniform" or "progress_estimate")')
 parser.add_argument('--algo', choices=('ppo', 'dqn'), default='ppo')
 parser.add_argument('--seed', default=None, type=int)
 parser.add_argument('-s', '--steps', type=float, default=6e6,
-                    help='Length of training in steps (default: 6e6).')
+    help='Length of training in steps (default: 6e6).')
 parser.add_argument('-w', '--wandb', action='store_true',
-                    help='Use wandb for analytics.')
-
+    help='Use wandb for analytics.')
+parser.add_argument('--project', default=None,
+    help='[Entity and] project for wandb. Eg: "safelife/multiagent" or "multiagent"')
+parser.add_argument('-d', '--depth', default=1, type=int,
```

Comment:
My aim is to get this merged into master soonish, and I'd argue for keeping the depth stuff off of the main branch as it seems mostly experimental and, as you point out in the help, isn't super well-supported.

Reply:
OK, I've reverted these commits in this branch. We could also rebase them out of existence with a bit more work, if desired.